_posts/2019-06-01-offene-unfallstatistiken.md

```diff
@@ -14,18 +14,18 @@ keywords:
   - "Verkehrsunfallstatistik"
 ---
 
-Seit geraumer Zeit beschäftigen wir uns bei Code for Münster mit der Verkehrsunfallstatistik von Münster.
+Seit geraumer Zeit beschäftigen wir uns bei Code for Münster mit der
+Verkehrsunfallstatistik von Münster.
 
 ## Alle Unfälle in Münster
 
 Zu jedem Verkehrsunfall, zu dem die Polizei hinzu gerufen wird, wird ein
-Datensatz mit den "Eckdaten" des Unfalls angelegt. Dazu gehört neben der Art
+Datensatz mit den „Eckdaten“ des Unfalls angelegt. Dazu gehört neben der Art
```

Comment:
Würde vorschlagen Anführungsstriche beim Standard zu belassen, falls dir die Darstellung nicht gefällt, kannst du gerne nochmal separat einen Pull request stellen in dem das per CSS gelöst wird.

Reply:
Die „normalen“ in der deutschen Sprache hab ich verwendet, sind leider nur nicht so leicht zu tippen mit den meisten Tastaturlayouts. "Gerade" Anführungszeichen sind eigentlich in den meisten natürlichen Sprachen fehl am Platz. https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen

Reply:
Jo, genau und weil sie nicht so leicht zu tippen sind, ist es einfacher die Anführungszeichen per CSS zu verändern. (https://developer.mozilla.org/en-US/docs/Web/CSS/quotes)

Reply:
Habe es zurück geändert. Denke der einfachste Weg ist es mit kramdown deutsche Smartquotes automatisiert erzeugen zu lassen. Separater PR folgt.

Reply:
:+1: